### ISSUE TITLE
Is the repo actively maintained

### ISSUE THREAD

**Issue by d-ivashchuk** [chengsokdara/use-whisper#54]
Hey @chengsokdara! awesome job on writing this hook, I would absolutely love to try it out but I was not sure if this library is maintained anymore as there are couple of critical requests pending in issues but no response to them.

Are you still planning to maintain it or giving maintainer rights to anybody?

Thanks a lot for the great work!

**Comment by Jberma23** [comment]
I don't think it is. It's open-source so im going to fork it and try to address some of the issues here

**Comment by ashwin-maurya** [comment]
@Jberma23 Hey is it working for you, I am not getting the output as {blob: undefined, text: undefined}

**Comment by nawangwe** [comment]
Would really be nice if someone else picked up the responsibility.
